resume.tex

```diff
@@ -24,26 +24,23 @@
 %==============================================================================
 
 \resumeSection{Work Experience}{
-  \experience{Software Development Engineer II}{MetLife Inc.}{2017-Present}{
-    \item Led a small team in building a proof-of-concept for a ``next-phase`` application built on Spring Boot and Angular 4, which was in turn used as a prototype for new projects company-wide
-	\item Consulted for a team building an application using my prototype as a template for a MongoDB-based plan repository with a Material Design UI
-	\item Implemented a report automation system through which users customize scheduled reports for the underwriting team
-    \item Enforced best practices and design principles by conducting code reviews
-  }
-  
+  \experienceHeader{MetLife Tech University Professional Program}{MetLife Inc.}{2014}\\
```

Comment:
I prefer all the MetLife job titles grouped together. There's not any signal in splitting up your accomplishments by the particular job title, apart from the fact that the earliest ones are (in my opinion) the best written =)

Reply:
Surely it's a coincidence that the earliest ones are also the ones that you've edited before ;)

Reply:
I'll combine the main ML job titles, but keep the MTU things separate.  I also like the most-recent-first ordering, so I'm going to re-order after I merge.  Other than those two things, I agree :)